### DISCUSSION TITLE
Project board issue card dragging broken on Firefox

### DISCUSSION THREAD

**Discussion by JEnglandOhalo** [community/community#56740] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Firefox 112.0.1 64 bit on Fedora Linux 37 Workstation.

Dragging and dropping cards has broken for me on Firefox. It is working on Chrome though. On FF I'm able to select the cards and the cursor icon changes, but I can't drag them to another column.  This functionality was working a few days ago.

The only errors I can see in the JS console are for content security policy violations blocking the loading of `pagewrap.bundle.js` and `prepareInjection.js`  although I'm not sure how relevant that is.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by hrishikesh-kocharekar-dev** [comment]
Check for JavaScript errors: Although the content security policy violations you mentioned may not be directly related to the dragging and dropping issue, it's worth investigating any JavaScript errors in the browser console. Try to fix those errors if possible, as they may have unintended consequences on the page's functionality.

Test in Firefox Private Browsing mode: Launch Firefox in Private Browsing mode and navigate to the page where the dragging and dropping issue occurs. This mode disables extensions and custom settings, providing a clean environment to test if the issue is specific to your browser configuration.
or 
Disable browser extensions: Certain browser extensions or add-ons can interfere with the functionality of web applications. Temporarily disable any extensions you have installed on Firefox and check if the dragging and dropping functionality starts working again.

**Reply by JEnglandOhalo** [reply]
Good call! Disabling extensions fixed the problem. Thank you!

**Comment by thomthom** [comment]
Also not working for me. Firefox 135.0.1.

I've tried with all extensions disabled. No errors in Dev Tools.

**Comment by afandian** [comment]
Drag and dropping an issue between columns on project view jumps around all over the place.

No log messages. 

Firefox 138.0 on Ubuntu, no extensions.
